OpenAI/Packages/com.openai.unity/Runtime/Common/ResponseFormat.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Runtime.Serialization;
+
+namespace OpenAI
+{
+    public enum ResponseFormat
+    {
+        [EnumMember(Value = "text")]
+        Text,
+        [EnumMember(Value = "json_object")]
+        Json
```

Comment:
Is ResponseFormat `auto` no longer supported?
I get the following error during AssistantRequest endpoint:

```ArgumentException: Could not cast or convert from System.String to OpenAI.ResponseFormatObject.```

Example JSON:
`{
  "object": "list",
  "data": [
    {
      "id": "...",
      "object": "assistant",
      "created_at": ...,
      "name": "...",
      "description": "...",
      "model": "gpt-4o",
      "instructions": "...",
      "tools": [],
      "top_p": 1.0,
      "temperature": 1.0,
      "tool_resources": {},
      "metadata": {},
      "response_format": "auto"
    }
  ]
}`

Reply:
Turns out this was if I retrieved an _existing_ assistant which had the response_format set to 'auto'. 
I've deleted the assistant for now to avoid the above issue, as it does look like `auto` isn't part of the v2 assistant documentation.

Reply:
> Turns out this was if I retrieved an existing assistant 

Do you remember which version it was?

Reply:
It was also a v2 Assistant. So interestingly, it does seem to be possible to set it to `auto` :)